docker/terra-node/Dockerfile

```diff
@@ -1,16 +1,26 @@
-# use node as base image
-FROM node:8.9.2
+FROM node:8
 
-# Prepare Directory for Source Code
-ENV APP_HOME /app
-RUN mkdir $APP_HOME
-WORKDIR $APP_HOME
+EXPOSE 8080
 
-# Mount Source into Container
-ONBUILD ADD . /app
+COPY ./entrypoint /opt/entrypoint
 
-# expose the port
-EXPOSE 8080
+RUN npm config set unsafe-perm true -g
```

Comment:
I forget the reason, why do we need to set unsafe-perm?

Reply:
It allows you to install packages as root (it does not affect all packages)
https://docs.npmjs.com/misc/config#unsafe-perm